File: 1751-2000/2000. Reverse Prefix of Word/solution.java
class Solution {
    public String reversePrefix(String word, char ch) {
        StringBuilder rs = new StringBuilder() ;
        boolean flag = true ; 
        for(char c : word.toCharArray() ){
            rs.append( c ) ; 
            if( c == ch && flag  ){
                rs = rs.reverse() ;
                flag = false ; 
            }
        }
        return rs.toString() ;
    }
}

File: 1751-2000/2000. Reverse Prefix of Word/solution.py
class Solution(object):
    def reversePrefix(self, word, ch):
        i = word.find(ch) 
        s = word[:i+1]
        t = ""
        if i+1 < len( word ) :
            t = word[i+1:]
        return s[::-1] + t 

        